### PR TITLE
[Cloud Posture] EKS rule templates

### PR DESCRIPTION
**Pull request by oren-zohar**
## What does this PR do?
Adding EKS benchmark rule templates

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] ~I have verified that all data streams collect metrics or logs.~
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] ~I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).~

## Author's Checklist
- [x] Waiting for https://github.com/elastic/integrations/pull/3654

## Related issues
- Closes: https://github.com/elastic/security-team/issues/4129

### CODE REVIEW

File: packages/cloud_security_posture/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.0.20"
```
Comment (kfirpeled) [inline packages/cloud_security_posture/changelog.yml:2]: nit we haven't promoted version `0.0.19` consider added the changes to that version backlog

see example [here](https://github.com/elastic/integrations/pull/3709/)

File: packages/cloud_security_posture/kibana/csp_rule_template/0434fc0b-4b72-5a73-8bee-8c8c40345165.json [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
+      "section": "API Server",
+      "version": "1.0",
+      "tags": [
+        "CIS",
+        "Kubernetes",
+        "CIS 1.2.18",
+        "API Server"
+      ],
+      "benchmark": {
+        "name": "CIS Kubernetes V1.23",
```
Comment (kfirpeled) [inline packages/cloud_security_posture/kibana/csp_rule_template/0434fc0b-4b72-5a73-8bee-8c8c40345165.json:27]: missing benchmark id `"cis_k8s"` - maybe you are missing a merge from main?

File: packages/cloud_security_posture/manifest.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 format_version: 1.0.0
 name: cloud_security_posture
 title: "CIS Kubernetes Benchmark"
-version: 0.0.18
+version: 0.0.20
```
Comment (kfirpeled) [inline packages/cloud_security_posture/manifest.yml:4]: consider using version 0.0.19 instead

File: packages/cloud_security_posture/changelog.yml (hunk 1 above)
Comment (oren-zohar) [reply packages/cloud_security_posture/changelog.yml:2]: @orouz's [PR](https://github.com/elastic/integrations/pull/3654) will add it, as seen in the description, I'm waiting for it to be merged

File: packages/cloud_security_posture/kibana/csp_rule_template/98524674-3de3-5424-9afa-ca763edca62a.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            "impact": "Some cloud providers may require this flag to ensure that hostname matches names issued by the cloud provider. In these environments, this recommendation should not apply.\n",
+            "default_value": "See the Amazon EKS documentation for the default value.\n",
+            "references": "1. https://kubernetes.io/docs/admin/kubelet/\n2. https://github.com/kubernetes/kubernetes/issues/22063\n3. https://kubernetes.io/docs/tasks/administer-cluster/reconfigure-kubelet/\n",
+            "section": "Kubelet",
+            "tags": [
```
Comment (kfirpeled) [inline packages/cloud_security_posture/kibana/csp_rule_template/98524674-3de3-5424-9afa-ca763edca62a.json:19]: missing `version`

File: packages/cloud_security_posture/kibana/csp_rule_template/d4eef2c6-599a-57d1-99dd-490d3745161d.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+            "impact": "Setting this parameter to `0` could result in a denial of service condition due to excessive events being created. The cluster's event processing and storage systems should be scaled to handle expected event loads.\n",
+            "default_value": "See the Amazon EKS documentation for the default value.\n",
+            "references": "1. https://kubernetes.io/docs/admin/kubelet/ 2. https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/kubeletconfig/v1beta1/types.go 3. https://kubernetes.io/docs/tasks/administer-cluster/reconfigure-kubelet/\n",
+            "section": "Kubelet",
+            "tags": [
```
Comment (kfirpeled) [inline packages/cloud_security_posture/kibana/csp_rule_template/d4eef2c6-599a-57d1-99dd-490d3745161d.json:19]: missing `version`

**Review by kfirpeled — COMMENTED** [review]
missing `version` field

**Review by kfirpeled — APPROVED** [review]
works for me locally
however we still have the same formatting issues like we had before
https://github.com/elastic/security-team/issues/3672

**Pull request merged**